pandas/tests/api/test_api.py

```diff
@@ -220,6 +221,22 @@ def test_api(self):
             self.ignored,
         )
 
+    def test_depr(self):
+        deprecated = (
+            self.deprecated_modules
+            + self.deprecated_classes
+            + self.deprecated_classes_in_future
+            + self.deprecated_funcs
+            + self.deprecated_funcs_in_future
+        )
+        for depr in deprecated:
+            with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
```

Comment:
Is the `check_stacklevel=False` needed here?

Reply:
Also, I would maybe add a separate, specific test for the numpy deprecation as well, where eg you can call `pd.np.arange(10)` and check that it actually works.

Reply:
fixed